app/controllers/placements/support/providers_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Placements::Support::ProvidersController < Placements::Support::ApplicationController
+  def new
+    @provider = Provider.new
+  end
+
+  def create
+    @provider = Provider.find_or_initialize_by(provider_params)
```

Comment:
Not sure if we need find or initialize here. I feel like we always want to create a new record in the `create` controller method

Reply:
resolved in [abdfae3](https://github.com/DFE-Digital/itt-mentor-services/pull/48/commits/abdfae3e4ab05b2c4cf0a45a0c634fec1fbf6545)